wpilibc/athena/src/Task.cpp

```diff
@@ -51,7 +51,7 @@ std::thread::native_handle_type Task::native_handle() {
  * @return true on success.
  */
 bool Task::Verify() {
-  TASK id = (TASK)m_thread.native_handle();
+  TASK id = reinterpret_cast<TASK>(m_thread.native_handle());
```

Comment:
I know. I was just trying to determine whether changing TASK to pthread_t would affect us porting this to other platforms.